discovery_child_development/pipeline/labelling/detection_management/prompts/examples.jsonl

```diff
@@ -0,0 +1,16 @@
+{"label": ["Detection", "Management"], "text": "Household income and maternal education in early childhood and activity-limiting chronic health conditions in late childhood: findings from birth cohort studies from six countries. Background We examined absolute and relative relationships between household income and maternal education during early childhood (&lt;5 years) with activity-limiting chronic health conditions (ALCHC) during later childhood in six longitudinal, prospective cohorts from high-income countries (UK, Australia, Canada, Sweden, Netherlands, USA). Methods Relative inequality (risk ratios, RR) and absolute inequality (Slope Index of Inequality) were estimated for ALCHC during later childhood by maternal education categories and household income quintiles in early childhood. Estimates were adjusted for mother ethnicity, maternal age at birth, child sex and multiple births, and were pooled using meta-regression. Results Pooled estimates, with over 42 000 children, demonstrated social gradients in ALCHC for high maternal education versus low (RR 1.54, 95% CI 1.28 to 1.85) and middle education (RR 1.24, 95% CI 1.11 to 1.38); as well as for high household income versus lowest (RR 1.90, 95% CI 1.66 to 2.18) and middle quintiles (RR 1.34, 95% CI 1.17 to 1.54). Absolute inequality showed decreasing ALCHC in all cohorts from low to high education (range: −2.85% Sweden, −13.36% Canada) and income (range: −1.8% Sweden, −19.35% Netherlands). Conclusion We found graded relative risk of ALCHC during later childhood by maternal education and household income during early childhood in all cohorts. Absolute differences in ALCHC were consistently observed between the highest and lowest maternal education and household income levels across cohort populations. Our results support a potential role for generous, universal financial and childcare policies for families during early childhood in reducing the prevalence of activity limiting chronic conditions in later childhood."}
```

Comment:
Hey Jess, thanks for this - yes, the idea is to cast a somewhat wide net that also includes environmental and social factors that can influence development. For example, some kind of policy to support household income of parents could be a promising social innovation for supporting child development, and so we want to find papers about topics like that as well.

In this case, I labelled this example as "both" as it in a way says income can be a proxy (or predictive of) future development outcomes - and it suggests a solution to manage low income (eg, generous childcare policies).

Hope it makes sense?

Reply:
Also thanks for flagging - I realised I need to change the "label" element to "Both" in these cases (rather than ["Detection", "Management"]) as that's the output from the OpenAI classifier function.